container/container.vars.make

```diff
@@ -1,3 +1,4 @@
 DOCKER := $(shell command -v docker || echo /usr/bin/docker)
 DOCKER_CONFIG := $(shell echo "$$HOME/.docker/config.json")
 DOCKER_COMPOSE := $(shell command -v docker-compose || echo /usr/local/bin/docker-compose)
+DOCKER_COMPOSE_VERSION := 1.29.2
```

Comment:
I would rename this to `DOCKER_COMPOSE_RELEASE` and then prefix the version with `tags/` so we can defer it to `latest` at a later time.